helpers/constants.py

```diff
@@ -0,0 +1,3 @@
+AddressZero = "0x0000000000000000000000000000000000000000"
```

Comment:
just saying there is `brownie.ZERO_ADDRESS`

Reply:
also redundant in this pr seems like..?

Reply:
Ah, didn't know about `brownie.ZERO_ADDRESS` nice! And you are right, not needed here, just copied the whole constants.py file from other repo while I was at it. Will remove the unnecessary ones.